doc/JSON_FLAGS.md

```diff
@@ -747,6 +747,7 @@ Some armor flags, such as `WATCH` and `ALARMCLOCK` are compatible with other ite
 
 - ```EATEN_HOT``` Morale bonus for eating hot.
 - ```EATEN_COLD``` Morale bonus for eating cold.
+- ```IRRADIATED``` Never spoils
```

Comment:
The whole idea of having irradiated as a flag is a bit of a hack. The most proper way to handle this would be to have a "sealed bag" container which would preserve contents, but become unsealed on access.

Will suffice for now, assuming it is handled correctly.